src/Hedgehog/Range.fs

```diff
@@ -20,8 +20,8 @@ module Range =
     let private bimap (f : 'a -> 'b) (g : 'c -> 'd) (a : 'a, b : 'c) : 'b * 'd =
         f a, g b
 
-    let map (f : 'a -> 'b) (Range (z, g) : Range<'a>) : Range<'b> =
-        Range (f z, fun sz ->
+    let map (f : 'a -> 'b) (Range (o, g) : Range<'a>) : Range<'b> =
+        Range (f o, fun sz ->
```

Comment:
This function could be moved down a little, and the `origin` function could be used to get the origin from the range. Less to change if the type definition changes.